test/unit/spec/ForgotPassword_spec.js

```diff
@@ -296,6 +296,28 @@ function (Q, Okta, OktaAuth, Util, AccountRecoveryForm, PrimaryAuthForm, Beacon,
             });
           });
       });
+      itp('should correct show back link in PwdResetEmailSent page', function () {
```

Comment:
`should hide back link in PwdResetEmailSent page when hideBackToSignInForReset is true`